content/articles/building-web-applications-with-devextreme/index.md

```diff
@@ -0,0 +1,119 @@
+ 3. ```npm run start```
+> The documentation for this can be found [here](https://js.devexpress.com/Documentation/Guide/React_Components/Create_a_DevExtreme_Application/)
+
+#### For Angular applications
+ 1. ```npx -p devextreme-cli devextreme new angular-app application-name```
+ 2. ```cd application-name```
+ 3. ```npm run start```
+> The documentation for this can be found [here](https://js.devexpress.com/Documentation/Guide/Angular_Components/Getting_Started/Create_a_DevExtreme_Application/)
+
+#### For Vue applications
+ 1. ```npx -p devextreme-cli devextreme new vue-app application-name```
+ 2. ```cd application-name```
+ 3. ```npm run start```
+> The documentation for this can be found [here](https://js.devexpress.com/Documentation/Guide/Vue_Components/Create_a_DevExtreme_Application/)
+
+### Building an Angular DevExtreme Application
+For demonstration purposes, I will create a DevExtreme application using my favorite JS framework, Angular.
+
+To create my Angular app, firstly I have to run:
+```
+npx -p devextreme-cli devextreme new angular-app my-extreme-app
+```
+After the installation, I have to cd into the project:
+```
+cd my-extreme-app
+```
+To start my application:
+```
+npm run start
+```
+To view your DevExtreme template, DevExtreme runs on a default port of 4200 which corresponds to the URL: http://localhost:4200. Visit that URL on your favorite browser after running the application. Always remember to go through the `ReadMe` file before starting the application in case of any change or update. 
+
+> The DevExtreme application template comes with drawer, data grid, and forms components.
+
+### Default Template View
+![default-template-view](/building-web-applications-with-devextreme/image1.png)
+
+### DevExtreme ThemeBuilder
+Manipulating DevExtreme is really easy. Using the *ThemeBuilder*, you can customize themes based on the DevExtreme default theme. The link to use the ThemeBuilder can be found [here](https://devexpress.github.io/ThemeBuilder/).
+
+To configure the template theme open `src\themes\metadata.additional.json` and `src\themes\metadata.base.json`. In the template, edit the theme name in the JSON file. By default, the theme is orange as you can see in the image above. After changing the theme color run  ```npm run build-themes``` or ```yarn build-themes``` for the change to take effect. I changed mine to blue in the two JSON files:
+
+```json
+{
+  "items": [],
+  "baseTheme": "material.blue.light",
+  "assetsBasePath": "../../../node_modules/devextreme/dist/css/",
+  "outputColorScheme": "base",
+  "base": true
+}
+```
+
+```json
+{
+ 
+  "items": [],
+  "baseTheme": "material.blue.dark",
+  "assetsBasePath": "../../../node_modules/devextreme/dist/css/",
+  "outputColorScheme": "additional",
+  "makeSwatch": true,
+  "base": true,
+  "widgets": [
+    "treeview",
+    "navbar"
+  ]
+
+}
+```
+Below is the result of the above changes:
+
+![blue-theme](/building-web-applications-with-devextreme/image2.png)
```

Comment:
The app looks slightly different than this on my end. I suppose it's likely just because of the version I'm using although it would be wise to ensure there aren't any other settings you are using that could make it appear this way.